stdlib/public/runtime/Casting.cpp

```diff
@@ -573,6 +577,9 @@ static void deallocateDynamicValue(OpaqueValue *value, const Metadata *type) {
       break;
       
     case ExistentialTypeRepresentation::Opaque:
+#ifdef SWIFT_RUNTIME_ENABLE_COW_EXISTENTIALS
+      assert(false && "Can't move out of copy-on-write boxes");
```

Comment:
`assert(false && …)` should generally be `swift_runtime_unreachable()` or `fatalError()`.